### PR TITLE
docs: add usage section to readme

### PR DESCRIPTION
**Pull request by MilanKovacic**
Added usage section to readme. Closes https://github.com/vitest-dev/vscode/issues/180.

image
image

### CODE REVIEW

**Comment by sheremet-va** [comment]
I don't really like the usage of a basic example in these screenshots. Maybe we can use an actual open-source project here?

**Comment by MilanKovacic** [comment]
My rationale for using the basic example was to not distract the users from the functionalities we are trying to explain.
I plan to update the main GIF showcasing the performance, and usability of the extension on a very large real world project.

What do you think?

File: README.md [hunk 1]
```diff
@@ -6,6 +6,42 @@
...
+- **Refresh Tests**: To reload your test suite, reflecting any new changes.
+- **Run All Tests**: To start testing all cases that are currently visible.
+- **Debug Tests**: To begin a debugging session for the tests.
+- **Continuous Run/Stop Continuous Run**: To toggle the watch mode for running tests on file changes.
+- **Show Output**: To display detailed logs from test executions.
+- **Miscellaneous Settings**: To customize the Testing view, such as sorting and grouping tests.
+
+The filter bar at the top of the Testing view allows you to narrow down the tests displayed, focusing on specific tests by name, exclusion patterns, or tags.
+
+Icons next to each test indicate their status—passed (checkmark), failed (cross), skipped (arrow), or not executed (dot).
```
Comment (sheremet-va) [inline README.md:30]: There is also yellow queued icon

File: README.md [hunk 2]
```diff
@@ -6,6 +6,45 @@
...
+
+![Testing view](./img/vitest-extension.png "Testing view")
+
+You can access the extension from the Testing view in the Visual Studio Code sidebar.
+
+The toolbar at the top provides various commands to manage test execution:
+
+- **Refresh Tests**: To reload your test suite, reflecting any new changes.
+- **Run All Tests**: To start testing all cases that are currently visible.
+- **Debug Tests**: To begin a debugging session for the tests.
```
Comment (sheremet-va) [inline README.md:23]: The button after debug tests is run tests with coverage

File: README.md (hunk 1 above)
Comment (MilanKovacic) [reply README.md]: Fixed.

**Review by sheremet-va — DISMISSED** [review]
Looks great! I really like it, thank you 🙏🏻

File: README.md (hunk 2 above)
Comment (MilanKovacic) [reply README.md:23]: Fixed.

**Review by sheremet-va — APPROVED** [review]

**Pull request merged**